tfk8s.go

```diff
@@ -89,13 +89,23 @@ func stripServerSideFields(m map[string]interface{}) {
 	}
 }
 
-func toHCL(doc map[interface{}]interface{}, providerAlias string, stripServerSide bool, mapOnly bool) (string, error) {
+func toHCL(doc map[interface{}]interface{}, providerAlias string, stripServerSide bool, mapOnly bool, defaultNamespace bool) (string, error) {
 	formattable := fixMap(doc)
 
 	if stripServerSide {
 		stripServerSideFields(formattable)
 	}
 
+	if defaultNamespace {
```

Comment:
I think instead of doing this mutation here, we can just add this conditional to [stripServerSideFields](https://github.com/jrhouston/tfk8s/blob/master/tfk8s.go#L74) and just not delete the namespace key. I think doing it this way will add a namespace to resources that are not namespaced?

Reply:
That was my first intent as well, but then I realized that `stripServerSideFields` is only being called when `-s` is passed on the CLI.
The two operations felt kind of orthogonal so I went like this. Do you mean I should drop the `-n` flag and just do this as part of `stripServerSideFields`?

Reply:
Well `-n` will only do something if `-s` is set anyway, because the default namespace always gets sent back and won't get stripped. 

I think maybe we can just remove the delete namespace line from stripServerSideFields